### PR TITLE
Add govuk pay team

### PR DESCRIPTION
**Pull request by heathd**

### CODE REVIEW

**Review by 36degrees — APPROVED** [review]
👍

**Comment by mrlumbu** [comment]
👍

**Review by alexbishop1 — APPROVED** [review]

**Comment by kbottla** [comment]
👍

File: config/alphagov.yml [hunk 1]
```diff
@@ -196,3 +196,30 @@ re-observe:
...
+    - heathd
+    - 46bit
+    - alexbishop1
+    - cobainc0
+    - DanailMinchev
+    - danworth
+    - ejrowley
+    - georgeracu
+    - heathd
+    - IreneLau
```
Comment (alexbishop1) [inline config/alphagov.yml]: It’s @IreneLau-GDS

**Review by alexbishop1 — DISMISSED** [review]

**Pull request merged**